wordpress-mobile/WordPress-Android
# Issue/13326 display scan finished message

This PR displays scan finished message to address p5T066-1OP#comment-7218.
(Strings matched with [iOS app](https://github.com/wordpress-mobile/WordPress-iOS/pull/15921)).

No threats | One threat | Multiple threats
--|--|--
 ![device-2021-02-22-114234](https://user-images.githubusercontent.com/1405144/108669719-94660e00-7503-11eb-994b-4ce6a33cb63d.png) | ![device-2021-02-22-113934](https://user-images.githubusercontent.com/1405144/108669706-8fa15a00-7503-11eb-9367-56f02683486b.png) | ![device-2021-02-22-114447](https://user-images.githubusercontent.com/1405144/108669729-992ac200-7503-11eb-8d38-7630dbcea067.png)

To test:

1. Launch the app
2. Open a site with Jetpack Scan enabled
3. Perform a scan where the result is no threats
4. Verify the notice appears with "no threats found"
5. Perform a scan where the result is 1 threat
6. Verify the notice appears with "1 potential threat found"
7. Perform a scan where the result is more than 1 threat
8. Verify the notice appears with "X potential threats found" where X is the number of threats

PR submission checklist:
